lib/activerecord-tenant-level-security/schema_statements.rb

```diff
@@ -1,22 +1,22 @@
 module TenantLevelSecurity
   module SchemaStatements
-    def create_policy(table_name)
+    def create_policy(table_name, column_name: 'tenant_id')
```

Comment:
We'll need to support this option in `CommandRecorder` and `SchemaDumper`.
https://github.com/kufu/activerecord-tenant-level-security/blob/v0.2.0/lib/activerecord-tenant-level-security/command_recorder.rb
https://github.com/kufu/activerecord-tenant-level-security/blob/v0.2.0/lib/activerecord-tenant-level-security/schema_dumper.rb

Reply:
Thank you for pointing that out. I've added it in these commit:
https://github.com/kufu/activerecord-tenant-level-security/pull/22/commits/9a588857ec996a2291b1dddd56d75199cabb3c5a
https://github.com/kufu/activerecord-tenant-level-security/pull/22/commits/7eafb3d01f2cb83763ca26dc79f9eac8ff495bf5